05.wc/wc.rb

```diff
@@ -0,0 +1,116 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  if File.pipe?($stdin)
+    print_standard(load_options)
+  else
+    print_argument(load_options)
```

Comment:
```suggestion
    print_standard
  else
    print_argument
```

ここも引数を渡す意味があまりないんじゃないかなーと思いました。

Reply:
以下のコミットで修正してます。

print_argumentではループ処理で引っかかるため、load_optionsを呼び出し、ARGVのオプション要素をなくす処理をしてます。

https://github.com/y-kawahara-gs/ruby-practices/pull/13/commits/0d01157965c13fdb87d4405a60b53e83b408e4e7